src/switch_core_media.c

```diff
@@ -9825,7 +9825,7 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_activate_rtp(switch_core_sessi
 											   a_engine->cur_payload_map->remote_sdp_port,
 											   a_engine->cur_payload_map->pt,
 											   a_engine->read_impl.samples_per_packet,
-											   a_engine->cur_payload_map->codec_ms * 1000,
```

Comment:
I don't get why the need to change this from microseconds to milliseconds? According to the switch_rtp_new, its expect:
```
   \param ms_per_packet time in microseconds per packet
```
If we will have to convert this param from microseconds to milliseconds, then we will have to update all of those who call switch_rtp_new and convert that milliseconds